### DISCUSSION TITLE
Failed to execute 'importScripts' on WorkerGlobalScope

### DISCUSSION THREAD

**Discussion by apple54whn** [ujjwalguptaofficial/JsStore#372] [Q&A]
Hello, I am using vite and incorporating jsstore through Web Worker. I want to utilize the Transaction feature. According to the documentation, I used `await connection.importScripts("./transaction.js");` to import it, but I encountered an error "Failed to execute 'importScripts' on WorkerGlobalScope". How should I modify this? 
thanks

**Comment by ujjwalguptaofficial** [comment]
Have you created transaction.js file ? make sure its accessible.
